readers/influxdb/messages_test.go

```diff
@@ -498,15 +498,15 @@ func TestReadJSON(t *testing.T) {
 
 	for desc, tc := range cases {
 		result, err := reader.ReadAll(tc.chanID, tc.pageMeta)
-
+		require.NoError(t, err)
+		
 		for i := 0; i < len(result.Messages); i++ {
 			m := result.Messages[i]
 			// Remove time as it is not sent by the client.
 			delete(m.(map[string]interface{}), "time")
 
 			result.Messages[i] = m
 		}
-		assert.Nil(t, err, fmt.Sprintf("%s: expected no error got %s", desc, err))
```

Comment:
Why is this assert removed?

Reply:
The assert checks the error after the loop but it should be done before just after getting the `result, err := reader.ReadAll(tc.chanID, tc.pageMeta)` .

Reply:
Is it done?